pkg/worker/handlers.go

```diff
@@ -24,23 +24,23 @@ type Handler struct {
 	// Raw address of overlord
 	overlordHost string
 	// Rooms map : RoomID -> Room
-	rooms map[string]*room.Room
+	rooms map[string]*room2.Room
```

Comment:
Why do you rename it to room2 and storage2?

Reply:
Ah. None. It was my refactoring tool. Will fix it. Thanks.

Reply:
What is your tool for lint. It's so magic. Teach me senpai.

Reply:
I normally use all the common go tools -
For lint and format:
```
- go fmt
- go import
- golangci-lint
```

For moving packages around I use `gomvpkg` from - https://github.com/golang/tools/tree/master/cmd

And for statci code check i use staticcheck. You will find the installations - `./hack/scripts/install_tools.sh`. And how to uses in the `Makefiles`.